src/explorer.ts

```diff
@@ -333,12 +333,15 @@ class KubernetesSelectorResource extends KubernetesResource {
         if (!this.selector) {
             return [];
         }
-        const selectors = [];
-        for (const sel in this.selector.matchLabels) {
-            selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
+        let selectorString = "";
+        if (this.selector.matchLabels) {
```

Comment:
@itowlson, realized after testing this morning that had missed porting over the logic for [pod selection](https://github.com/Azure/vscode-kubernetes-tools/blob/master/src/kubectlUtils.ts#L177) based on [labels](https://github.com/Azure/vscode-kubernetes-tools/blob/master/src/kubectlUtils.ts#L190) and wanted to bring attention to that. I can send an PR later today to fix this. :( sorry about missing those conditions.